optuna/visualization/_param_importances.py

```diff
@@ -81,6 +86,11 @@ def objective(trial):
             A list of names of parameters to assess.
             If :obj:`None`, all parameters that are present in all of the completed trials are
             assessed.
+        target:
+            A function to specify the value to evaluate importances. If it is :obj:`None`, the
+            objective values are used.
+        target_name:
+            Target's name to display on the axis label.
```

Comment:
We set this name as not the axis label but the title.
```suggestion
            Target's name to display on the title. The title would be "Importance for ``target_name``".
```

Reply:
It's the title of the xaxis. I think "axis label" makes sense.

Reply:
Ops! Sorry for the confusion. Let me resolve this conversation without any changes.